src/hooks/useListEventsFiltered.ts

```diff
@@ -10,6 +10,7 @@ export default function useListEventsFiltered(
   filter?: {
     search?: string | null
     type?: ToggleItemsValue | null
+    tag?: string | null
```

Comment:
we should use `categories` instead of `tags`
```suggestion
    categories?: string | null
```